app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -79,11 +79,10 @@ class SearchMapper {
     JsObject( Map( "$in" -> JsArray(values.toSeq.map(writes.writes)) ) )
   }
 
-  private def nullifyNoneValues: String => JsValue = { v: String =>
-    v match {
-      case "none" => JsNull
-      case _ => JsString(v)
-    }
+  private def mappingNoneOrSome: String => JsValue = {
+    case "none" => JsNull
+    case "some" => Json.obj("$ne" -> JsNull)
```

Comment:
potentially a problem... 
what if an assignee id or a queue id are called "some"?

Reply:
but the same could happen also with "none"

Reply:
Employee "some" will have to be let go.

Reply:
Assignee IDs and Queue IDs are numerical